vulcanai2/plotters/visualization.py

```diff
@@ -21,17 +20,56 @@
 import logging
 logger = logging.getLogger(__name__)
 
+def assert_display_available():
+    """
+    Check that display variable exists
+
+    Returns
+    -------
+    Boolean
+        True if display variable exists
+        False if display variable does not exist
+
+    """
+    return True if os.environ.get("DISPLAY") else False
 
-def display_record(record=None, load_path=None, interactive=True):
+def save_visualization(plot, path=None):
+    """
+    Save plot at designated path
+
+    Parameters
+    ----------
+    plot : matplotlib
+        Matplotlib variable with savefig ability.
+    path : string
+        String that designates the path to save the given figure to.
+
+    Returns
+    -------
+    None
+
+    """
+    if not assert_display_available() and path is None:
+        raise RuntimeError("No display environment found. Display environment needed to plot, or set save_path=path/to/dir")
+    else:
+        plot.savefig(path)
+
+def get_time():
+    curr_time = str(datetime.now())
+    curr_time = curr_time.replace(" ", '_')
+    curr_time = curr_time.split('.')[0]
+    return curr_time
```

Comment:
This is what we have for saving the models, could we standardize? 
  `self._save_path = "{}_{date:%Y-%m-%d_%H:%M:%S}/".format(
                self.name, date=datetime.now())`

Reply:
@nirtiac You got it!

Reply:
@nirtiac @rfratila The only issue I see with this is that `self.name` is not being sent to `visualization`.

Reply:
You could have name as the parameter of the function, as you do provide a name/label when you call get_time below

Reply:
@nirtiac Okay I see, that's what I just updated it to do. I'll condense to standardize

Reply:
do you still need get_time @joemehltretter ?